broadinstitute/pearl
# [JN-363] Added nightly fetch of kit status from DSM

No UI in this PR, so no screenshots. There is a service function for fetching the status of a single kit, but that's not hooked in yet.

Primarily, there's a scheduled service that runs nightly that updates the status of all in-progress kits.

To test:
1. Run the tests; verify that they pass.
2. Modify the timing in `ScheduledKitStatusService` so that it will run in the next couple of minutes.
3. Start the admin API and UI and open https://localhost:3000/ourhealth/studies/ourheart/env/sandbox/participants/OHSALK/kitRequests. You'll see the seeded kit with its fake status.
4. Wait for `ScheduledKitStatusService` to run; it will log start and finish.
5. Reload OHSALK's kit requests and see that the DSM status has updated.

There's also some iteration on the integration with DSM for sending kits. However, I haven't been able to make a successful request to DSM dev yet, so that's still work-in-progress.
